webpage/src/.vuepress/config.js

```diff
@@ -469,65 +469,67 @@ module.exports = {
           }
         ]
       },
-      '/fa/': {
-        selectText: 'زبان ها',
-        label: 'فارسی',
-        editLinkText: 'برای بهبود این سایت به ما کمک کنید!',
-        algolia: {
-          apiKey: '35f878f4b712d5ab6a659eb0a6c71576',
-          indexName: 'qownnotes'
-        },
-        nav: [
-          {
-            text: 'شروع شدن',
-            link: utils.getNavItemForLanguage(gettingStartedNavItem, 'fa'),
-          },
-          {
-            text: 'نصب و راه اندازی',
-            link: utils.getNavItemForLanguage(installationNavItem, 'fa'),
-          },
-          {
-            text: 'تغییرات',
-            link: changelogNavItem,
-            collapsable: true,
-          },
-          {
-            text: 'مخاطب',
-            ariaLabel: 'Contact Menu',
-            items: contactNavItems
-          },
-          {
-            text: 'اهدا کردن',
-            link: utils.getNavItemForLanguage(donateNavItem, 'fa'),
-          }
-        ],
-        sidebar: [
-          {
-            title: 'شروع شدن',
-            children: utils.getNavItemsForLanguage(gettingStartedNavItems, 'fa')
-          },
-          {
-            title: 'نصب و راه اندازی',
-            children: utils.getNavItemsForLanguage(installationNavItems, 'fa')
-          },
-          {
-            title: 'ویرایشگر',
-            children: utils.getNavItemsForLanguage(editorNavItems, 'fa')
-          },
-          {
-            title: 'اسکریپت ها',
-            children: utils.getNavItemsForLanguage(scriptingNavItems, 'fa')
-          },
-          {
-            title: 'مشارکت',
-            children: utils.getNavItemsForLanguage(contributingNavItems, 'fa')
-          },
-          {
-            title: 'سالات متداول',
-            children: utils.getNavItemsForLanguage(faqNavItems, 'fa')
-          }
-        ]
-      },
+      ‚/fa/': {
```

Comment:
All `'` characters seem to be broken, the indent is off too and there is a wrong `,` at the beginning of your changes.